components/atoms/TextInput/text-input.tsx

```diff
@@ -61,9 +61,9 @@ const TextInput = ({
             name={name}
             id={id || name || ""}
             placeholder={placeholder || ""}
-            className={`flex-1 focus:outline-none  ${classNames} ${
-              disabled && "bg-light-slate-3 cursor-not-allowed  text-light-slate-9"
-            }`}
+            className={`flex-1 focus:outline-none  ${
```

Comment:
We don't need to bind `classNames` here.  This is an intended change or a misintend???

Reply:
It is an intentional change to prevent input from inheriting the containers style. We might need expanding more styles props moving forward